cluster/redis-clustering.gemspec

```diff
@@ -47,5 +47,5 @@ Gem::Specification.new do |s|
   s.required_ruby_version = '>= 2.7.0'
 
   s.add_runtime_dependency('redis', s.version)
-  s.add_runtime_dependency('redis-cluster-client', '>= 0.7.0')
+  s.add_runtime_dependency('redis-cluster-client', '>= 0.7.0', "< 0.7.10")
```

Comment:
Why `< 0.7.10` ?

If we need a new release of it, it can wait. cc @supercaracal

Reply:
`0.8.0` has released.

* https://github.com/redis-rb/redis-cluster-client/releases/tag/v0.8.0
* https://rubygems.org/gems/redis-cluster-client/versions/0.8.0

Reply:
`0.7.11` was included some changes for the transaction feature. We can get rid of the restriction if the following PR is merged.

* #1255

Reply:
@byroot yeah sorry I was waiting for that specific fix for the watch command, hence the pinning.
I just rebased my branch, and PR is updated.